emitter/emitter.go

```diff
@@ -687,3 +694,22 @@ func emitMovementStatement(movementStmt *ast.MovementStatement) string {
 	sb.WriteString(fmt.Sprintf("\t%s\n", terminator))
 	return sb.String()
 }
+
+func emitMartStatement(martStmt *ast.MartStatement) string {
+	terminator := "ITEM_NONE"
+	var sb strings.Builder
+	sb.WriteString("\t.align 2\n")
+	if martStmt.Scope == token.GLOBAL {
+		sb.WriteString(fmt.Sprintf("%s::\n", martStmt.Name.Value))
+	} else {
+		sb.WriteString(fmt.Sprintf("%s:\n", martStmt.Name.Value))
+	}
+	for _, item := range martStmt.MartItems {
+		if item == terminator {
+			break
+		}
+		sb.WriteString(fmt.Sprintf("\t.2byte %s\n", item))
+	}
+	sb.WriteString("\t.2byte ITEM_NONE\n\trelease\n\tend\n")
```

Comment:
Is `release, end` actually needed in pokeemerald/pokefirered?  As far as I can tell, the script engine stops reading when it hits a `0` (`ITEM_NONE`).

Reply:
I have no idea. However, the user is probably expecting the output to resemble vanilla and would get confused/ask questions if it didn't, so I would probably keep it in just for user confidence reasons. It's not really _hurting_ anything, after all...

Reply:
They are not needed (they're never reached, as you say it stops at the first `0` while reading items)